hack/release.toml

```diff
@@ -4,6 +4,7 @@ commit = "HEAD"
 project_name = "terraform-provider-talos"
 github_repo = "siderolabs/terraform-provider-talos"
 match_deps = "^github.com/(siderolabs/[a-zA-Z0-9-]+)$"
+ignore_deps = ["github.com/siderolabs/talos"]
```

Comment:
added this, otherwise changelog pulls in all changes from talos since initial commit, not sure if we need it

Reply:
or another option is to move action tracker in `pkg/machinery` not sure if we need that